modules/lightning_features/lightning_media/config/schema/lightning_media.schema.yml

```diff
@@ -35,5 +35,13 @@ entity_browser.browser.widget.file_upload:
       type: boolean
       label: 'Return underlying file entity'
 
-field.widget.settings.image_immutable:
-  type: field.widget.settings.image_image
+field.widget.third_party.lightning_media:
+  type: mapping
```

Comment:
Turns out, you can inject third-party settings into field widget and formatter configuration! I had no idea you could do this, and it's super-useful, but the settings must have a config schema defined. Thus, this.